src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
+
+## Mobile
+
+<ImageGrid images={[Specs1Image]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
+
+<ImageGrid images={[Specs2Image]} regularWidth />
+<Divider />
+
+# Developers
+
+We recommend using the following components to build this pattern:
+
+**Angular**:
+
+-   [@angular/material/toolbar](https://material.angular.io/components/toolbar)
+
+**React**:
+
+-   [@material-ui/core/AppBar](https://material-ui.com/api/app-bar/)
+-   [@material-ui/core/Toolbar](https://material-ui.com/api/toolbar/)
+
+**React Native**:
+
+-   [@pxblue/react-native-components/Header](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--header)
```

Comment:
I would like for us to come up with a more interesting way to present this information instead of a bulleted list...maybe another cool callout card of some kind - we can build a shareable component.

Reply:
Where's Ionic?

Reply:
Change it to Angular & Ionic?

Reply:
> to come up with a more interesting way to present this information instead of a bulleted list
I can add this into the backlog as part of the design overhauls... our docs are already full of bullet lists, so perhaps it is fine here?

Reply:
> Change it to Angular & Ionic?

It's been a while, I can't remember if we use the mat-toolbar for Ionic apps or if we use the Ionic toolbar...it might be a mix.

Reply:
> I can add this into the backlog as part of the design overhauls... our docs are already full of bullet lists, so perhaps it is fine here?

That's fine, I just think bulleted lists are boring for this kind of information (and they take up a bunch of space when we're showing stuff for all four frameworks at a time).

Reply:
Added to the backlog as PXBLUE-1513 for the next grooming